### ISSUE TITLE
Default values for invest input fields

### ISSUE THREAD

**Issue by davemfish** [natcap/invest-workbench#153]
Right now all input fields are blank when a model is opened (unless you're opening a "recent run"). There are some inputs where it always makes sense to populate a default value. 

Examples:
* rec model "start year" and "end year"
* urban cooling "adjust * weight" inputs
* habitat quality "half-saturation constant"
* probably others I didn't think of

Now that we have a sample data download feature, we could default to populating all fields with the sample data if it's available for that model. That solves this problem, and it achieves another feature on our todo list (easy "one-click" loading of sample data).

If sample data is not available - it was an optional download - should we still implement something that can populate just these constants with default values? If so would that be something to add to `ui_config.js`? I might be okay with leaving all the fields blank if the sample data is unavailable.

**Comment by dcdenu4** [comment]
>I might be okay with leaving all the fields blank if the sample data is unavailable.

I tend to agree with this. I'd be interested in Stacie's thoughts as well. To me, if a user isn't loading sample data, then they should have to think about the values they are entering instead of using something blindly.

**Comment by emlys** [comment]
I'd be in favor of separate "populate with sample data" and "show a default value" features. As a user I don't always want the form to be filled out right away. I think that should be opt-in with something like a button that auto-fills all the fields.

I think any default values should be set up on the workbench side in `ui_config.js` like you suggest

**Comment by davemfish** [comment]
> I'd be in favor of separate "populate with sample data" and "show a default value" features. As a user I don't always want the form to be filled out right away. I think that should be opt-in with something like a button that auto-fills all the fields.

It sounds reasonable to keep offering the blank form as an option then, along with a load sample data button, or something. I'd suggest that if we do have default values for any args, they should just show up, like by default 😃 

>To me, if a user isn't loading sample data, then they should have to think about the values they are entering instead of using something blindly.

I think this applies to most cases, like maybe that HQ saturation constant, but for others like the Rec model "start and end year", I think it very much makes sense to have a default that the user can use without thinking about it. So maybe that HQ constant shouldn't be on this list at all. And maybe there aren't very many args at all that should be given defaults.

Other items that should have defaults are all the options `lists` (soon to be `set/dict` in the new args spec) (dropdowns). Since they currently default to whatever is first in the list (I assume) it makes sense to control that more carefully, probably in `ui_config.js`.

**Comment by emlys** [comment]
> I'd suggest that if we do have default values for any args, they should just show up, like by default 😃

Agree! If the defaults are just one or two fields, they should show up right away. 

> I think this applies to most cases, like maybe that HQ saturation constant, but for others like the Rec model "start and end year", I think it very much makes sense to have a default that the user can use without thinking about it. So maybe that HQ constant shouldn't be on this list at all. And maybe there aren't very many args at all that should be given defaults.

I agree that defaults should be reserved for those few cases where there's one, obviously right value that will apply to most cases. I don't think we need to go looking for default values where they're not jumping out.

**Comment by davemfish** [comment]
A forum user just suggested populating default values for some SDR parameters:

>Would be nice if the model calibration parameters came with the default values pre-loaded, for example in the SDR model with Borselli, MaxSDR and IC0 with 2/0.8/0.5 values already filled at launch.

**Comment by davemfish** [comment]
on hold until we migrate this repo to natcap/invest
